docusaurus/docs/importing-a-component.md

```diff
@@ -48,3 +48,28 @@ Learn more about ES6 modules:
 - [When to use the curly braces?](https://stackoverflow.com/questions/36795819/react-native-es-6-when-should-i-use-curly-braces-for-import/36796281#36796281)
 - [Exploring ES6: Modules](http://exploringjs.com/es6/ch_modules.html)
 - [Understanding ES6: Modules](https://leanpub.com/understandinges6/read#leanpub-auto-encapsulating-code-with-modules)
+
+## Absolute Imports
+
+You can configure your application to support importing modules using absolute paths. This can be done by configuring a `jsconfig.json` or `tsconfig.json` file in the root of your project. If you're using TypeScript in your project, you will already have the `tsconfig.json` file present.
+
+Below is an example `jsconfig.json` file for a JavaScript project. You can create the file if it doesn't already exist:
+
+```json
+{
+  "compilerOptions": {
+    "baseUrl": "src"
+  },
+  "include": ["src/**/*"]
```

Comment:
I don't think this is needed. I could definitely be wrong about that though. When I was testing this I wasn't using it.

Reply:
You definitely don't _need_ it, you're right. It's recommended by VS Code team to avoid performance issues when you add this file. This link explains it well: https://code.visualstudio.com/docs/languages/jsconfig. The key thing to note:

> If you do not have a jsconfig.json in your workspace, VS Code will by default exclude the node_modules folder.

> When your JavaScript project is growing too large and performance slows, it is often because of library folders like node_modules. If VS Code detects that your project is growing too large, it will prompt you to edit the exclude list.

So once we add this file, it by default will include everything in the directory of `jsconfig.json` and all subdirectories, **including** `node_modules`. By using `include` it explicitly says "only check these files, thanks!", which is equivalent to how we have our configuration for `tsconfig.json`.

Thoughts?

Reply:
That seems like a pretty good reason to me.